test/test-energinet-data-api-download.py

```diff
@@ -0,0 +1,68 @@
+#!/usr/bin/env python
```

Comment:
I don't think any energinet stuff should be here, maybe renaming to something like test-download-from-browser-locally.py?